INFO 2025-04-21 23:32:47,664 autoreload Watching for file changes with StatReloader
ERROR 2025-04-21 23:33:10,343 basehttp "GET / HTTP/1.1" 500 59
ERROR 2025-04-21 23:33:14,242 basehttp "GET /favicon.ico HTTP/1.1" 500 59
INFO 2025-04-21 23:36:26,288 autoreload Watching for file changes with StatReloader
WARNING 2025-04-21 23:36:35,398 log Not Found: /
WARNING 2025-04-21 23:36:35,399 basehttp "GET / HTTP/1.1" 404 2644
WARNING 2025-04-21 23:36:35,543 log Not Found: /favicon.ico
WARNING 2025-04-21 23:36:35,543 basehttp "GET /favicon.ico HTTP/1.1" 404 2695
INFO 2025-04-21 23:36:44,241 basehttp "GET /admin HTTP/1.1" 301 0
INFO 2025-04-21 23:36:44,246 basehttp "GET /admin/ HTTP/1.1" 302 0
INFO 2025-04-21 23:36:44,272 basehttp "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4198
INFO 2025-04-21 23:36:44,280 basehttp "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2025-04-21 23:36:44,280 basehttp "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2025-04-21 23:36:44,281 basehttp "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2025-04-21 23:36:44,326 basehttp "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2025-04-21 23:36:44,326 basehttp "GET /static/admin/css/login.css HTTP/1.1" 304 0
INFO 2025-04-21 23:36:44,326 basehttp "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2025-04-21 23:36:44,329 basehttp "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2025-04-21 23:36:49,854 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2025-04-21 23:36:49,865 basehttp "GET /admin/ HTTP/1.1" 200 5515
INFO 2025-04-21 23:36:49,884 basehttp "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2025-04-21 23:36:49,885 basehttp "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2025-04-21 23:36:49,885 basehttp "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2025-04-21 23:38:18,352 basehttp "POST /api/v1/auth/login/ HTTP/1.1" 200 566
INFO 2025-04-21 23:38:31,170 basehttp "POST /api/v1/auth/login/ HTTP/1.1" 200 566
ERROR 2025-04-21 23:38:33,224 log Internal Server Error: /api/v1/tenants/
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 503, in dispatch
    self.initial(request, *args, **kwargs)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 421, in initial
    self.check_permissions(request)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 339, in check_permissions
    self.permission_denied(
    ~~~~~~~~~~~~~~~~~~~~~~^
        request,
        ^^^^^^^^
        message=getattr(permission, 'message', None),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        code=getattr(permission, 'code', None)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 181, in permission_denied
    raise exceptions.PermissionDenied(detail=message, code=code)
rest_framework.exceptions.PermissionDenied: 您没有执行该操作的权限。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\module_loading.py", line 16, in cached_import
    return getattr(module, class_name)
AttributeError: module 'common.exceptions' has no attribute 'custom_exception_handler'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 179, in import_from_string
    return import_string(val)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\module_loading.py", line 32, in import_string
    raise ImportError(
    ...<2 lines>...
    ) from err
ImportError: Module "common.exceptions" does not define a "custom_exception_handler" attribute/class

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 469, in handle_exception
    exception_handler = self.get_exception_handler()
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 304, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 168, in perform_import
    return import_from_string(val, setting_name)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 182, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'common.exceptions.custom_exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: Module "common.exceptions" does not define a "custom_exception_handler" attribute/class.
ERROR 2025-04-21 23:38:33,229 basehttp "POST /api/v1/tenants/ HTTP/1.1" 500 136902
INFO 2025-04-21 23:38:53,374 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:38:53,701 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:39:00,790 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:39:07,586 basehttp "POST /api/v1/auth/login/ HTTP/1.1" 200 566
INFO 2025-04-21 23:41:49,525 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:41:49,637 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:41:49,841 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:41:49,949 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:42:08,426 basehttp "POST /api/v1/auth/login/ HTTP/1.1" 200 566
ERROR 2025-04-21 23:42:10,490 log Internal Server Error: /api/v1/users/me/
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 503, in dispatch
    self.initial(request, *args, **kwargs)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 421, in initial
    self.check_permissions(request)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 339, in check_permissions
    self.permission_denied(
    ~~~~~~~~~~~~~~~~~~~~~~^
        request,
        ^^^^^^^^
        message=getattr(permission, 'message', None),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        code=getattr(permission, 'code', None)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 180, in permission_denied
    raise exceptions.NotAuthenticated()
rest_framework.exceptions.NotAuthenticated: 身份认证信息未提供。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\module_loading.py", line 16, in cached_import
    return getattr(module, class_name)
AttributeError: module 'common.exceptions' has no attribute 'custom_exception_handler'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 179, in import_from_string
    return import_string(val)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\module_loading.py", line 32, in import_string
    raise ImportError(
    ...<2 lines>...
    ) from err
ImportError: Module "common.exceptions" does not define a "custom_exception_handler" attribute/class

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 469, in handle_exception
    exception_handler = self.get_exception_handler()
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 304, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 168, in perform_import
    return import_from_string(val, setting_name)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\settings.py", line 182, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'common.exceptions.custom_exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: Module "common.exceptions" does not define a "custom_exception_handler" attribute/class.
ERROR 2025-04-21 23:42:10,509 basehttp "GET /api/v1/users/me/ HTTP/1.1" 500 137050
INFO 2025-04-21 23:42:35,882 autoreload D:\GitHub\integrated_project\common\exceptions\__init__.py changed, reloading.
INFO 2025-04-21 23:42:36,192 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:42:47,317 autoreload D:\GitHub\integrated_project\common\authentication\jwt_auth.py changed, reloading.
INFO 2025-04-21 23:42:47,629 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:42:47,965 autoreload D:\GitHub\integrated_project\common\authentication\jwt_auth.py changed, reloading.
INFO 2025-04-21 23:42:48,276 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:43:00,457 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:43:00,765 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:43:00,822 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:43:01,128 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:43:07,914 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:43:08,225 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:43:08,270 autoreload D:\GitHub\integrated_project\core\settings.py changed, reloading.
INFO 2025-04-21 23:43:08,584 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:43:13,998 autoreload Watching for file changes with StatReloader
INFO 2025-04-21 23:43:26,981 basehttp "POST /api/v1/auth/login/ HTTP/1.1" 200 566
INFO 2025-04-21 23:43:29,034 basehttp "GET /api/v1/users/me/ HTTP/1.1" 200 299
INFO 2025-04-21 23:44:04,614 basehttp "GET /api/v1/docs/ HTTP/1.1" 200 4667
ERROR 2025-04-21 23:44:05,641 log Internal Server Error: /api/v1/schema/
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\views.py", line 84, in get
    return self._get_schema_response(request)
           ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\views.py", line 92, in _get_schema_response
    data=generator.get_schema(request=request, public=self.serve_public),
         ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\generators.py", line 285, in get_schema
    paths=self.parse(request, public),
          ~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\generators.py", line 256, in parse
    operation = view.schema.get_operation(
        path, path_regex, path_prefix, method, self.registry
    )
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 112, in get_operation
    operation['responses'] = self._get_response_bodies()
                             ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1397, in _get_response_bodies
    return {'200': self._get_response_for_code(response_serializers, '200', direction=direction)}
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1453, in _get_response_for_code
    component = self.resolve_serializer(serializer, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1648, in resolve_serializer
    component.schema = self._map_serializer(serializer, direction, bypass_extensions)
                       ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 949, in _map_serializer
    schema = self._map_basic_serializer(serializer, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1048, in _map_basic_serializer
    schema = self._map_serializer_field(field, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 684, in _map_serializer_field
    component = self.resolve_serializer(field, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1648, in resolve_serializer
    component.schema = self._map_serializer(serializer, direction, bypass_extensions)
                       ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 949, in _map_serializer
    schema = self._map_basic_serializer(serializer, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1042, in _map_basic_serializer
    for field in serializer.fields.values():
                 ^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\serializers.py", line 375, in fields
    fields[key] = value
    ~~~~~~^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\utils\serializer_helpers.py", line 167, in __setitem__
    field.bind(field_name=key, parent=self.serializer)
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\fields.py", line 358, in bind
    assert self.source != field_name, (
           ^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: It is redundant to specify `source='display_name'` on field 'CharField' in serializer 'UserMinimalSerializer', because it is the same as the field name. Remove the `source` keyword argument.
ERROR 2025-04-21 23:44:05,645 basehttp "GET /api/v1/schema/ HTTP/1.1" 500 202876
ERROR 2025-04-21 23:44:05,674 log Internal Server Error: /api/v1/schema/
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 515, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 486, in raise_uncaught_exception
    raise exc
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\views.py", line 84, in get
    return self._get_schema_response(request)
           ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\views.py", line 92, in _get_schema_response
    data=generator.get_schema(request=request, public=self.serve_public),
         ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\generators.py", line 285, in get_schema
    paths=self.parse(request, public),
          ~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\generators.py", line 256, in parse
    operation = view.schema.get_operation(
        path, path_regex, path_prefix, method, self.registry
    )
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 112, in get_operation
    operation['responses'] = self._get_response_bodies()
                             ~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1397, in _get_response_bodies
    return {'200': self._get_response_for_code(response_serializers, '200', direction=direction)}
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1453, in _get_response_for_code
    component = self.resolve_serializer(serializer, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1648, in resolve_serializer
    component.schema = self._map_serializer(serializer, direction, bypass_extensions)
                       ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 949, in _map_serializer
    schema = self._map_basic_serializer(serializer, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1048, in _map_basic_serializer
    schema = self._map_serializer_field(field, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 684, in _map_serializer_field
    component = self.resolve_serializer(field, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1648, in resolve_serializer
    component.schema = self._map_serializer(serializer, direction, bypass_extensions)
                       ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 949, in _map_serializer
    schema = self._map_basic_serializer(serializer, direction)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\drf_spectacular\openapi.py", line 1042, in _map_basic_serializer
    for field in serializer.fields.values():
                 ^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\serializers.py", line 375, in fields
    fields[key] = value
    ~~~~~~^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\utils\serializer_helpers.py", line 167, in __setitem__
    field.bind(field_name=key, parent=self.serializer)
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\rest_framework\fields.py", line 358, in bind
    assert self.source != field_name, (
           ^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: It is redundant to specify `source='display_name'` on field 'CharField' in serializer 'UserMinimalSerializer', because it is the same as the field name. Remove the `source` keyword argument.
ERROR 2025-04-21 23:44:05,676 basehttp "GET /api/v1/schema/ HTTP/1.1" 500 202876
INFO 2025-04-22 09:21:56,715 autoreload Watching for file changes with StatReloader
WARNING 2025-04-22 09:22:02,202 log Not Found: /
WARNING 2025-04-22 09:22:02,202 basehttp "GET / HTTP/1.1" 404 2644
WARNING 2025-04-22 09:22:02,415 log Not Found: /favicon.ico
WARNING 2025-04-22 09:22:02,415 basehttp "GET /favicon.ico HTTP/1.1" 404 2695
INFO 2025-04-22 09:22:07,596 basehttp "GET /admin HTTP/1.1" 301 0
INFO 2025-04-22 09:22:07,602 basehttp "GET /admin/ HTTP/1.1" 302 0
INFO 2025-04-22 09:22:07,616 basehttp "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4198
INFO 2025-04-22 09:22:07,777 basehttp "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
INFO 2025-04-22 09:22:07,781 basehttp "GET /static/admin/js/theme.js HTTP/1.1" 200 1653
INFO 2025-04-22 09:22:07,782 basehttp "GET /static/admin/css/base.css HTTP/1.1" 200 22154
INFO 2025-04-22 09:22:07,838 basehttp "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
INFO 2025-04-22 09:22:07,838 basehttp "GET /static/admin/css/login.css HTTP/1.1" 200 951
INFO 2025-04-22 09:22:07,838 basehttp "GET /static/admin/css/responsive.css HTTP/1.1" 200 16632
INFO 2025-04-22 09:22:07,845 basehttp "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
INFO 2025-04-22 09:22:09,907 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4355
INFO 2025-04-22 09:22:12,479 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4355
INFO 2025-04-22 09:22:15,201 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2025-04-22 09:22:15,216 basehttp "GET /admin/ HTTP/1.1" 200 5515
INFO 2025-04-22 09:22:15,225 basehttp "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
INFO 2025-04-22 09:22:15,237 basehttp "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
INFO 2025-04-22 09:22:15,237 basehttp "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
INFO 2025-04-22 09:22:21,336 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 7247
INFO 2025-04-22 09:22:21,347 basehttp "GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
INFO 2025-04-22 09:22:21,348 basehttp "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
INFO 2025-04-22 09:22:21,349 basehttp "GET /static/admin/js/core.js HTTP/1.1" 200 6208
INFO 2025-04-22 09:22:21,350 basehttp "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9777
INFO 2025-04-22 09:22:21,352 basehttp "GET /static/admin/js/actions.js HTTP/1.1" 200 8076
INFO 2025-04-22 09:22:21,353 basehttp "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
INFO 2025-04-22 09:22:21,353 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:22:21,353 basehttp "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
INFO 2025-04-22 09:22:21,354 basehttp "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2025-04-22 09:22:21,355 basehttp "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
INFO 2025-04-22 09:22:21,356 basehttp "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
INFO 2025-04-22 09:22:21,358 basehttp "GET /static/admin/js/filters.js HTTP/1.1" 200 978
INFO 2025-04-22 09:22:21,369 basehttp "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2025-04-22 09:33:24,553 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:33:33,770 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:33:33,787 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:33:34,755 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:33:34,768 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:33:35,497 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:33:35,510 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:33:36,934 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 17344
INFO 2025-04-22 09:33:36,946 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:33:37,055 basehttp "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
INFO 2025-04-22 09:33:37,056 basehttp "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
INFO 2025-04-22 09:33:39,016 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:33:39,029 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:33:39,890 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:33:39,905 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:33:41,611 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:45,282 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:45,852 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:46,452 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:46,653 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:46,839 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:47,035 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:33:48,979 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 17344
INFO 2025-04-22 09:33:48,991 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:34:04,524 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:34:04,539 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:34:05,826 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 17344
INFO 2025-04-22 09:34:05,839 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:34:10,378 basehttp "GET /admin/common/ HTTP/1.1" 200 5376
INFO 2025-04-22 09:34:11,211 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:37:39,856 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:37:44,091 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:37:45,128 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:37:45,654 basehttp "GET /admin/ HTTP/1.1" 200 6475
WARNING 2025-04-22 09:37:49,835 log Not Found: /
WARNING 2025-04-22 09:37:49,836 basehttp "GET / HTTP/1.1" 404 2644
INFO 2025-04-22 09:37:55,203 basehttp "GET /admin HTTP/1.1" 301 0
INFO 2025-04-22 09:37:55,210 basehttp "GET /admin/ HTTP/1.1" 200 6475
INFO 2025-04-22 09:38:05,632 basehttp "GET /admin/auth/group/ HTTP/1.1" 200 8038
INFO 2025-04-22 09:38:05,644 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:38:09,743 basehttp "GET /admin/auth/group/add/ HTTP/1.1" 200 12327
INFO 2025-04-22 09:38:09,760 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:38:09,897 basehttp "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
INFO 2025-04-22 09:38:09,900 basehttp "GET /static/admin/js/SelectBox.js HTTP/1.1" 200 4530
INFO 2025-04-22 09:38:09,903 basehttp "GET /static/admin/css/forms.css HTTP/1.1" 200 8525
INFO 2025-04-22 09:38:09,905 basehttp "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
INFO 2025-04-22 09:38:09,905 basehttp "GET /static/admin/css/widgets.css HTTP/1.1" 200 11991
INFO 2025-04-22 09:38:09,906 basehttp "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 15709
INFO 2025-04-22 09:38:09,935 basehttp "GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291
INFO 2025-04-22 09:44:53,102 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:44:59,733 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:01,485 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:02,234 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:02,657 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:03,090 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:05,556 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:07,979 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:15,145 basehttp "GET /admin/ HTTP/1.1" 200 6494
WARNING 2025-04-22 09:45:17,677 log Not Found: /
WARNING 2025-04-22 09:45:17,678 basehttp "GET / HTTP/1.1" 404 2644
INFO 2025-04-22 09:45:22,043 basehttp "GET /admin/ HTTP/1.1" 200 6494
INFO 2025-04-22 09:45:28,251 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 17363
INFO 2025-04-22 09:45:28,264 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:47:02,768 autoreload D:\GitHub\integrated_project\users\admin.py changed, reloading.
INFO 2025-04-22 09:47:03,060 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:47:14,171 autoreload D:\GitHub\integrated_project\users\admin.py changed, reloading.
INFO 2025-04-22 09:47:14,460 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:47:28,600 autoreload D:\GitHub\integrated_project\tenants\admin.py changed, reloading.
INFO 2025-04-22 09:47:28,883 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:47:39,164 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:47:44,500 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 19574
INFO 2025-04-22 09:47:44,518 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:47:49,091 basehttp "GET /admin/tenants/tenant/ HTTP/1.1" 200 12138
INFO 2025-04-22 09:47:49,106 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:47:52,564 basehttp "GET /admin/tenants/tenant/add/ HTTP/1.1" 200 15470
INFO 2025-04-22 09:47:52,576 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:48:06,634 autoreload D:\GitHub\integrated_project\core\admin.py changed, reloading.
INFO 2025-04-22 09:48:06,958 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:48:20,988 basehttp "POST /admin/tenants/tenant/add/ HTTP/1.1" 302 0
INFO 2025-04-22 09:48:21,014 basehttp "GET /admin/tenants/tenant/add/ HTTP/1.1" 200 15717
INFO 2025-04-22 09:48:21,027 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:48:21,108 basehttp "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2025-04-22 09:48:27,474 basehttp "GET /admin/users/user/ HTTP/1.1" 200 16353
INFO 2025-04-22 09:48:27,489 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:48:32,850 basehttp "GET /admin/users/user/add/ HTTP/1.1" 200 13966
INFO 2025-04-22 09:48:32,860 basehttp "GET /static/admin/css/unusable_password_field.css HTTP/1.1" 200 663
INFO 2025-04-22 09:48:32,864 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 09:48:32,870 basehttp "GET /static/admin/js/unusable_password_field.js HTTP/1.1" 200 1480
INFO 2025-04-22 09:49:08,521 autoreload D:\GitHub\integrated_project\tenants\admin.py changed, reloading.
INFO 2025-04-22 09:49:08,824 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:49:15,809 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:52:34,139 autoreload D:\GitHub\integrated_project\common\admin.py changed, reloading.
INFO 2025-04-22 09:52:34,440 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:52:59,742 autoreload D:\GitHub\integrated_project\tenants\admin.py changed, reloading.
INFO 2025-04-22 09:53:00,033 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:53:20,213 autoreload D:\GitHub\integrated_project\users\admin.py changed, reloading.
INFO 2025-04-22 09:53:20,505 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 09:53:42,769 autoreload D:\GitHub\integrated_project\common\admin.py changed, reloading.
INFO 2025-04-22 09:53:43,057 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 10:41:49,770 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 10:42:00,901 autoreload D:\GitHub\integrated_project\users\admin.py changed, reloading.
INFO 2025-04-22 10:42:01,196 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 10:42:01,351 autoreload D:\GitHub\integrated_project\tenants\admin.py changed, reloading.
INFO 2025-04-22 10:42:01,647 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 10:42:06,560 basehttp "GET /admin/users/user/add/ HTTP/1.1" 200 15664
INFO 2025-04-22 10:42:06,577 basehttp "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,577 basehttp "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,577 basehttp "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,578 basehttp "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,579 basehttp "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,580 basehttp "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,580 basehttp "GET /static/admin/js/core.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,581 basehttp "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,581 basehttp "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,582 basehttp "GET /static/admin/js/actions.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,583 basehttp "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,583 basehttp "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,583 basehttp "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,584 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:06,596 basehttp "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,596 basehttp "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:42:06,608 basehttp "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:42:12,815 basehttp "GET /admin/users/user/add/ HTTP/1.1" 200 15664
INFO 2025-04-22 10:42:12,827 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:14,354 basehttp "GET /admin/auth/group/add/ HTTP/1.1" 200 14557
INFO 2025-04-22 10:42:14,366 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:14,408 basehttp "GET /static/admin/img/search.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:42:16,940 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:42:16,956 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:29,226 basehttp "GET /admin/tenants/tenant/add/ HTTP/1.1" 200 15470
INFO 2025-04-22 10:42:29,240 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:33,501 basehttp "GET /admin/users/user/ HTTP/1.1" 200 16353
INFO 2025-04-22 10:42:33,511 basehttp "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
INFO 2025-04-22 10:42:33,514 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:33,522 basehttp "GET /static/admin/js/filters.js HTTP/1.1" 304 0
INFO 2025-04-22 10:42:33,522 basehttp "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:42:45,418 basehttp "GET /admin/tenants/tenantquota/ HTTP/1.1" 200 13407
INFO 2025-04-22 10:42:45,430 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:42:54,228 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:42:54,244 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
ERROR 2025-04-22 10:43:01,377 log Internal Server Error: /admin/tenants/tenantquota/add/
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\mysql\base.py", line 76, in execute
    return self.cursor.execute(query, args)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\MySQLdb\cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\MySQLdb\cursors.py", line 330, in _query
    db.query(q)
    ~~~~~~~~^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\MySQLdb\connections.py", line 280, in query
    _mysql.connection.query(self, query)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
MySQLdb.IntegrityError: (1048, "Column 'tenant_id' cannot be null")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\contrib\admin\options.py", line 719, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\contrib\admin\options.py", line 1984, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\contrib\admin\options.py", line 1843, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\contrib\admin\options.py", line 1894, in _changeform_view
    self.save_model(request, new_object, form, not add)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\GitHub\integrated_project\common\admin.py", line 45, in save_model
    super().save_model(request, obj, form, change)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\contrib\admin\options.py", line 1314, in save_model
    obj.save()
    ~~~~~~~~^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\base.py", line 902, in save
    self.save_base(
    ~~~~~~~~~~~~~~^
        using=using,
        ^^^^^^^^^^^^
    ...<2 lines>...
        update_fields=update_fields,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\base.py", line 1008, in save_base
    updated = self._save_table(
        raw,
    ...<4 lines>...
        update_fields,
    )
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\base.py", line 1169, in _save_table
    results = self._do_insert(
        cls._base_manager, using, fields, returning_fields, raw
    )
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\base.py", line 1210, in _do_insert
    return manager._insert(
           ~~~~~~~~~~~~~~~^
        [self],
        ^^^^^^^
    ...<3 lines>...
        raw=raw,
        ^^^^^^^^
    )
    ^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\query.py", line 1854, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\models\sql\compiler.py", line 1881, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\django\db\backends\mysql\base.py", line 76, in execute
    return self.cursor.execute(query, args)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\MySQLdb\cursors.py", line 179, in execute
    res = self._query(mogrified_query)
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\MySQLdb\cursors.py", line 330, in _query
    db.query(q)
    ~~~~~~~~^^^
  File "C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.13_qbz5n2kfra8p0\LocalCache\local-packages\Python313\site-packages\MySQLdb\connections.py", line 280, in query
    _mysql.connection.query(self, query)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
django.db.utils.IntegrityError: (1048, "Column 'tenant_id' cannot be null")
ERROR 2025-04-22 10:43:01,385 basehttp "POST /admin/tenants/tenantquota/add/ HTTP/1.1" 500 250516
INFO 2025-04-22 10:43:09,929 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:11,557 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:11,569 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:43:11,969 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:11,984 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:43:12,204 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:12,216 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:43:12,402 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:12,428 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:43:12,593 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:12,618 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:43:12,807 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 15830
INFO 2025-04-22 10:43:12,826 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:45:04,888 autoreload D:\GitHub\integrated_project\tenants\admin.py changed, reloading.
INFO 2025-04-22 10:45:05,185 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 10:46:09,989 autoreload Watching for file changes with StatReloader
INFO 2025-04-22 10:46:25,834 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 17068
INFO 2025-04-22 10:46:25,850 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:46:25,866 basehttp "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:46:35,114 basehttp "GET /admin/tenants/tenant/add/?_to_field=id&_popup=1 HTTP/1.1" 200 9095
INFO 2025-04-22 10:46:35,126 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:46:39,900 basehttp "GET /admin/users/user/ HTTP/1.1" 200 17656
INFO 2025-04-22 10:46:39,913 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:46:39,923 basehttp "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:46:50,323 basehttp "GET /admin/users/user/3/change/ HTTP/1.1" 200 31169
INFO 2025-04-22 10:46:50,336 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:46:50,421 basehttp "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
INFO 2025-04-22 10:46:50,424 basehttp "GET /static/admin/js/calendar.js HTTP/1.1" 200 9141
INFO 2025-04-22 10:46:50,459 basehttp "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
INFO 2025-04-22 10:46:50,459 basehttp "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
INFO 2025-04-22 10:47:05,940 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 19962
INFO 2025-04-22 10:47:05,953 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:09,193 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 19962
INFO 2025-04-22 10:47:09,208 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:09,760 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 19962
INFO 2025-04-22 10:47:09,774 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:09,987 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 19962
INFO 2025-04-22 10:47:10,005 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:10,205 basehttp "GET /admin/common/apilog/ HTTP/1.1" 200 19962
INFO 2025-04-22 10:47:10,220 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:32,490 basehttp "GET /admin HTTP/1.1" 301 0
INFO 2025-04-22 10:47:32,499 basehttp "GET /admin/ HTTP/1.1" 200 9627
INFO 2025-04-22 10:47:32,550 basehttp "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 304 0
WARNING 2025-04-22 10:47:32,666 log Not Found: /favicon.ico
WARNING 2025-04-22 10:47:32,667 basehttp "GET /favicon.ico HTTP/1.1" 404 2695
INFO 2025-04-22 10:47:34,474 basehttp "GET /admin/tenants/tenant/ HTTP/1.1" 200 15145
INFO 2025-04-22 10:47:34,489 basehttp "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,491 basehttp "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,492 basehttp "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,493 basehttp "GET /static/admin/js/core.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,494 basehttp "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,495 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:34,495 basehttp "GET /static/admin/js/actions.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,496 basehttp "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,496 basehttp "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,497 basehttp "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,497 basehttp "GET /static/admin/img/search.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,500 basehttp "GET /static/admin/js/filters.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,506 basehttp "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:47:34,507 basehttp "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:47:35,960 basehttp "GET /admin/tenants/tenantquota/ HTTP/1.1" 200 13890
INFO 2025-04-22 10:47:35,973 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:39,953 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 17068
INFO 2025-04-22 10:47:39,961 basehttp "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:39,961 basehttp "GET /static/admin/css/forms.css HTTP/1.1" 304 0
INFO 2025-04-22 10:47:39,964 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:39,973 basehttp "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
INFO 2025-04-22 10:47:39,973 basehttp "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
INFO 2025-04-22 10:47:45,816 basehttp "POST /admin/tenants/tenantquota/add/ HTTP/1.1" 200 17307
INFO 2025-04-22 10:47:45,828 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:49,756 basehttp "GET /admin/tenants/tenantquota/ HTTP/1.1" 200 13890
INFO 2025-04-22 10:47:49,768 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:55,051 basehttp "POST /admin/tenants/tenantquota/ HTTP/1.1" 200 10264
INFO 2025-04-22 10:47:55,060 basehttp "GET /static/admin/js/cancel.js HTTP/1.1" 200 884
INFO 2025-04-22 10:47:56,033 basehttp "POST /admin/tenants/tenantquota/ HTTP/1.1" 302 0
INFO 2025-04-22 10:47:56,042 basehttp "GET /admin/tenants/tenantquota/ HTTP/1.1" 200 11256
INFO 2025-04-22 10:47:56,057 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:47:56,063 basehttp "GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
INFO 2025-04-22 10:47:57,983 basehttp "GET /admin/tenants/tenantquota/add/ HTTP/1.1" 200 17068
INFO 2025-04-22 10:47:57,994 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:48:00,850 basehttp "POST /admin/tenants/tenantquota/add/ HTTP/1.1" 302 0
INFO 2025-04-22 10:48:00,860 basehttp "GET /admin/tenants/tenantquota/ HTTP/1.1" 200 13620
INFO 2025-04-22 10:48:00,874 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:48:03,816 basehttp "GET /admin/tenants/tenantquota/ HTTP/1.1" 200 13407
INFO 2025-04-22 10:48:03,829 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:48:04,535 basehttp "GET /admin/tenants/tenant/ HTTP/1.1" 200 15145
INFO 2025-04-22 10:48:04,548 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:48:06,024 basehttp "GET /admin/users/user/ HTTP/1.1" 200 17656
INFO 2025-04-22 10:48:06,037 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
WARNING 2025-04-22 10:48:33,438 log Not Found: /
WARNING 2025-04-22 10:48:33,438 basehttp "GET / HTTP/1.1" 404 2644
INFO 2025-04-22 10:49:01,395 basehttp "GET /admin/ HTTP/1.1" 200 10656
INFO 2025-04-22 10:49:01,414 basehttp "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2025-04-22 10:49:01,447 basehttp "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
WARNING 2025-04-22 10:49:01,573 log Not Found: /favicon.ico
WARNING 2025-04-22 10:49:01,574 basehttp "GET /favicon.ico HTTP/1.1" 404 2695
INFO 2025-04-22 10:49:04,088 basehttp "GET /admin/users/user/ HTTP/1.1" 200 17656
INFO 2025-04-22 10:49:04,100 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:49:08,238 basehttp "GET /admin/users/user/add/ HTTP/1.1" 200 15716
INFO 2025-04-22 10:49:08,247 basehttp "GET /static/admin/css/forms.css HTTP/1.1" 304 0
INFO 2025-04-22 10:49:08,247 basehttp "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
INFO 2025-04-22 10:49:08,250 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:49:08,254 basehttp "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
INFO 2025-04-22 10:49:08,254 basehttp "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
INFO 2025-04-22 10:49:22,053 basehttp "POST /admin/users/user/add/ HTTP/1.1" 200 16069
INFO 2025-04-22 10:49:22,064 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:49:34,484 basehttp "POST /admin/users/user/add/ HTTP/1.1" 200 15994
INFO 2025-04-22 10:49:34,494 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:49:48,118 basehttp "POST /admin/users/user/add/ HTTP/1.1" 200 15961
INFO 2025-04-22 10:49:48,130 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:50:04,583 basehttp "POST /admin/users/user/add/ HTTP/1.1" 200 15967
INFO 2025-04-22 10:50:04,595 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
INFO 2025-04-22 10:50:20,715 basehttp "POST /admin/users/user/add/ HTTP/1.1" 302 0
INFO 2025-04-22 10:50:20,727 basehttp "GET /admin/users/user/add/ HTTP/1.1" 200 15942
INFO 2025-04-22 10:50:20,742 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 9327
